src/Illuminate/Auth/Events/Failed.php

```diff
@@ -19,14 +19,24 @@ class Failed
     public $credentials;
 
     /**
+     * The request.
+     *
+     * @var \Illuminate\Http\Request
+     */
+    protected $request;
+
+    /**
      * Create a new event instance.
      *
      * @param  \Illuminate\Contracts\Auth\Authenticatable|null  $user
      * @param  array  $credentials
+     * @param  \Illuminate\Http\Request  $request
+     * @return void
      */
-    public function __construct($user, $credentials)
+    public function __construct($user, $credentials, $request)
```

Comment:
Thx for the catch @laurencei - changed that in https://github.com/laravel/framework/pull/20674/commits/d75b31efed31f5b3556dd1fd8b21869dfa65b2c5